build-tools/xa-prep-tasks/Xamarin.Android.BuildTools.PrepTasks/AcceptAndroidSdkLicenses.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+namespace Xamarin.Android.BuildTools.PrepTasks
+{
+	public class AcceptAndroidSdkLicenses : Task
+	{
+		[Required]
+		public string AndroidSdkDirectory { get; set; }
+
+		public override bool Execute ()
+		{
+			var licdir = Path.Combine (Path.Combine (AndroidSdkDirectory, "licenses"));
+			if (!Directory.Exists (licdir))
```

Comment:
This `if` isn't necessary; `Directory.CreateDirectory()` doesn't throw if the directory already exists.